### ISSUE TITLE
Some links of Lean recipes in README are broken due to inconsistent naming

### ISSUE THREAD

**Issue by utensil** [cpitclaudel/alectryon#82]
Some links of Lean recipes in README are broken due to inconsistent naming:

> See `<recipes/plain-lean4.lean>`__, `<recipes/lean4-tactics.rst>`__, `<recipes/lean4-tactics-myst.md>`__ and `<recipes/literate-lean4.lean>`__  for examples.

Only the first link works, the rest are broken because the actual file used underscores instead of hyphens.

Links in the Lean 3 section are OK as far as I've checked.

P.S. Could it be consistent whether the lean3/4 is the prefix? It's better if they are all prefix, so they are listed together in the directory.

**Comment by cpitclaudel** [comment]
Thanks a lot for spotting this.  Could you / Would you like to prepare a PR?

**Comment by utensil** [comment]
Yes I can, and I guess they can be renamed to using underscore to keep consistency with other files (like the coq ones) .

P.S. `lean4-tactics*` are no longer there, so languages are all postfixes.

**Comment by cpitclaudel** [comment]
Fixed.
